app/password/new/template.hbs

```diff
@@ -0,0 +1,58 @@
+{{#login-box}}
+  <div class="row">
+    <div class="col-xs-6 col-xs-offset-3">
+
+      <div class="panel panel-default panel-subdued panel-focused">
+        <div class="panel-heading">
+          <div class="panel-heading-link">
+            {{link-to "Sign in instead" "login"}}
+          </div>
+          <h3>Reset your password</h3>
+        </div>
+
+        <div class="panel-body">
+          {{#if error}}
+            <div class="alert alert-danger fade in">
+              {{error}}
+            </div>
+          {{else}}
+            {{#if hasSubmitted}}
+              {{#if errors.model.password}}
+                {{#bs-alert alert="warning"}}
+                  Password {{join-array errors.model.password ', '}}.
+                {{/bs-alert}}
+              {{/if}}
+              {{#if errors.model.passwordConfirmation}}
+                {{#bs-alert alert="warning"}}
+                  Confirmation {{errors.model.passwordConfirmation}}.
+                {{/bs-alert}}
```

Comment:
These are validation, not API errors. They don't make a request, I don't believe there is any base.

Can definitely use some DRY, though I'd like to do it separately.